### ISSUE TITLE
Character Editor : Weapon floats at spawn

### ISSUE THREAD

**Issue by ghramsey** [MattFiler/OpenCAGE#216]
I started with a fresh commands.pak and tried experimenting with character spawn again.
Spawned a shotgun character from \human\security\grunt_npc_shotgun .  Set position near start of M3.
When the character spawns with their shot gun and moves the first time the shotgun separates and stays floating in air.
Upon killing them the weapon they're holding vanishes but the weapon in air remains floating.
I did same thing with \human\security\grunt_npc_pistol and spawned them near the first.  The pistol does same thing
it separates from where they were spawned and remains in air.

I made two video clips.  They are not previewing correctly in github preview mode.  
IDK if they're working right or just need to be processed before they work.

Here's same videos on YT:

https://youtu.be/96ClcX2ayig
https://youtu.be/YSrfN5had6w

*Didn't think those direct uploads worked. I'll just remove them.*

**Comment by MattFiler** [comment]
This is an interesting one!

**Comment by ghramsey** [comment]
Yeah. I didn't go through all the interation of NPC entities to see if it does the same with riot guard for example.
I'll test with those later.

**Comment by MattFiler** [comment]
Did you have any luck with any others?

**Comment by ghramsey** [comment]
No.  I didn't go through them all yet.
